src/actions/setup.ts

```diff
@@ -47,9 +47,16 @@ export async function setupAction(input: SetupInput = setupInput()) {
   if (!input.token) {
     info(`Skipped authentication: 'token' not provided.`);
   } else {
-    await group('Validating authenticated account', () =>
-      authenticate(input.token, input.easVersion ? 'eas' : input.expoVersion ? 'expo' : undefined)
-    );
+    await group('Validating authenticated account', async () => {
+      if (input.easVersion) {
+        authenticate(input.token, 'eas');
+      }
+      if (input.expoVersion) {
```

Comment:
Is this a refactor or a logical change?

Reply:
I would call that logical change, locally when I try to run there was an error and this would be the fix, now, I might be wrong.